### PR TITLE
Complete report

### PR DESCRIPTION
**Pull request by ibalejandro**
Connecting the reports to the case detail and presenting the information and possible interactions in the correct format.

### CODE REVIEW

File: src/components/ConductCard/ConductCard.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const conductChangeHandler = (event) => {
+        onConductChange(event.target.value);
+    };
+
+    return (
+        <Card className={readOnly ? "mb-4" : "mb-4 mt-4"}>
+            <Card.Header><strong>{cardHeader}</strong></Card.Header>
+            <Card.Body>
+                <Form>
+                    <Form.Group controlId="exampleForm.ControlTextarea1">
```
Comment (juanto121) [inline src/components/ConductCard/ConductCard.js:16]: exampleForm.ControlTextarea1?

File: src/services/ReportsAPI.js [hunk 2]
```diff
@@ -26,15 +56,143 @@ class ReportsAPI {
...
+            }
+        }
+        if (symptomsList.length === 0) {
+            symptomsList.push("N/A");
+        }
+        return symptomsList;
+    }
+
+    static getComorbidities(report) {
+        const diagnosedWith = {...report.diagnosedWith};
```
Comment (juanto121) [inline src/services/ReportsAPI.js:154]: list is not being modified so a new object is not required

File: src/services/ReportsAPI.js [hunk 3]
```diff
@@ -26,15 +56,143 @@ class ReportsAPI {
...
+            isolationReason = "CONTAGIO DE COVID-19";
+        }
+        else {
+            isolationReason = "POR ORDEN DEL GOBIERNO";
+        }
+        return isolationReason;
+    }
+
+    static getSubmissionDate(report) {
+        const submissionTimestamp = new Date(parseInt(report.submissionTimestamp));
```
Comment (juanto121) [inline src/services/ReportsAPI.js:188]: consider using yoda time

File: src/containers/CaseDetail/CaseDetail.js [hunk 4]
```diff
@@ -7,50 +7,148 @@ import UserDetailedSymptomsCard from "../../components/UserDetailedSymptomsCard/
...
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import styled from "styled-components";
 import {useAuth0} from "../../shared/Auth";
+import DoctorUserCommunication from "../DoctorUserCommunication/DoctorUserCommunication";
+import ConductCard from "../../components/ConductCard/ConductCard";
 
 const CaseDetail = () => {
     //TODO: LOAD FROM HISTORY PROPS THE PATH to get the caseId
```
Comment (juanto121) [inline src/containers/CaseDetail/CaseDetail.js:21]: Is this todo done?

File: src/containers/CaseDetail/CaseDetail.js [hunk 5]
```diff
@@ -7,50 +7,148 @@ import UserDetailedSymptomsCard from "../../components/UserDetailedSymptomsCard/
...
-        const loadCaseState = async (caseId) => {
-            const states = await CasesAPI.getStatesForCase(caseId)
+        const loadCaseState = async (id, token) => {
+            const states = await CasesAPI.getStatesForCase(id, token)
             setCaseState(states)
         }
-        loadCaseState()
-    }, [caseId])
+        loadCaseState(id, token)
+    }, [id])
```
Comment (juanto121) [inline src/containers/CaseDetail/CaseDetail.js:39]: should also depend on token and isAuthenticated. Check if token is not null before loadingCaseState

**Comment by juanto121** [comment]
Cambiar a paleta de colores:
https://color.adobe.com/es/docs-web-app-color-theme-14733388

File: src/components/ConductCard/ConductCard.js (hunk 1 above)
Comment (ibalejandro) [reply src/components/ConductCard/ConductCard.js:16]: No idea what the purpose of it was. Removed.

File: src/services/ReportsAPI.js (hunk 2 above)
Comment (ibalejandro) [reply src/services/ReportsAPI.js:154]: Correct. Same object used.

File: src/containers/CaseDetail/CaseDetail.js (hunk 4 above)
Comment (ibalejandro) [reply src/containers/CaseDetail/CaseDetail.js:21]: Yes, it is. Removed.

**Pull request merged**